<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ã˜sse er best</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>twitch.tv/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="streamComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="qualityComboBox">
       <item>
        <property name="text">
         <string>source</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>high</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>low</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mobile</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="processOutput">
     <property name="font">
      <font>
       <family>DejaVu Sans Mono</family>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
